NorthstarDLL/util/printmaps.cpp

```diff
@@ -182,10 +184,32 @@ void ConCommand_maps(const CCommand& args)
 			spdlog::info("({}) {}", PrintMapSource.at(map.source), map.name);
 }
 
+void ConCommand_map(const CCommand& args)
+{
+	if (args.ArgC() > 1)
+	{
+		RefreshMapList();
+
+		const char* arg = args.Arg(1);
+		auto f = [&](MapVPKInfo map) -> bool { return map.name == arg; };
```

Comment:
I assume a map is deemed invalid if its name isn't contained in the loaded maps names, am I right?

Reply:
Loading an invalid map ( can't be loaded for some reason ) causes an engine error